stock_orderpoint_automatic_creation/models/product_category.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Daniel Campos <danielcampos@avanzosc.es> - Avanzosc S.L.
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductCategory(models.Model):
+    _inherit = 'product.category'
+
+    create_orderpoints = fields.Selection(
```

Comment:
A strange choice: why not a simple checkbox? Usability is better: one click instead of 2
Any reason for this choice?